packages/store-ui/src/Minicart/Button/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Button } from 'theme-ui'
+import type { ComponentPropsWithoutRef, FC, ComponentPropsWithRef } from 'react'
+
+import MinicartButtonIcon from './Icon'
+import MinicartButtomBadge from './Badge'
+
+interface Props
+  extends ComponentPropsWithoutRef<typeof Button>,
```

Comment:
Use the Button props instead of it

import type { ButtonProps } from 'theme-ui'

Reply:
why so? doesn't it only make the api more complex to maintain and evolve? `typeof` already is enought, right ?

Reply:
Why make api more complex? I didn't catch this point of view

Reply:
Because each component export becomes two. One for the component itself and another for its props